src/main/scala/se/nimsa/sbx/box/BoxDAO.scala

```diff
@@ -40,7 +40,7 @@ class BoxDAO(val dbConf: DatabaseConfig[JdbcProfile])(implicit ec: ExecutionCont
 
   class BoxTable(tag: Tag) extends Table[Box](tag, BoxTable.name) {
     def id = column[Long]("id", O.PrimaryKey, O.AutoInc)
-    def name = column[String]("name")
+    def name = column[String]("name", O.Length(255))
```

Comment:
How does slick handle string columns if they don't have a length? Is there a default length, or is this dependent on the database implementation?